packages/rendermime/src/registry.ts

```diff
@@ -305,23 +305,41 @@ export namespace RenderMimeRegistry {
   }
 
   /**
-   * A default resolver that uses a session and a contents manager.
+   * A default resolver that uses a given reference path and a contents manager.
    */
   export class UrlResolver implements IRenderMime.IResolver {
     /**
-     * Create a new url resolver for a console.
+     * Create a new url resolver.
      */
     constructor(options: IUrlResolverOptions) {
-      this._session = options.session;
+      if (options.path) {
+        this._path = options.path;
+      } else if (options.session) {
+        this._session = options.session;
```

Comment:
Should this include a deprecation warning via `console.warn` ?

Reply:
Hmm, do we log deprecation warnings elsewhere?

Reply:
Saul, are you saying we *do* log deprecation warnings elsewhere, and we should here, or that it was a good question to ask?

Reply:
I'm not sure either, but we probably should have a policy on this. Good topic for dev call today.

Reply:
We decided not to have warnings for now in our meeting.